redbot/cogs/trivia/data/lists/elements.yaml

```
@@ -1,4 +1,5 @@
 AUTHOR: Mamalus
+DESCRIPTION: Can you identify the element symbol from the element name?
```

Maybe mention that it's about chemical elements? Maybe it's just me (I'm not a native English speaker) but it may not be immediately clear what "elements" this is about.
```suggestion
DESCRIPTION: Can you identify the symbol of a chemical element from its name?
```

---

I agree with this, thanks